src/Symfony/Component/Notifier/Bridge/FakeSms/CHANGELOG.md

```diff
@@ -0,0 +1,7 @@
+CHANGELOG
+=========
+
+5.3.0
+-----
+
+ * Added the bridge
```

Comment:
```suggestion
5.3
---

 * Add the bridge
```

See https://github.com/symfony/symfony-docs/pull/14830 for more informations

I've the same feedback on my PR ;)